README.md

```diff
@@ -42,6 +42,10 @@ Use the following settings to update cell execution time for a notification and
     // Settings for the Notifications extension
     // ****************************************
 
+    // Cell Number Type
+    // Type of cell number to display when the report_cell_number is true. Select from 'cell_index' or ‘cell_execution_count'.
+    "cell_number_type": "cell_index",
+
```

Comment:
I'd add this below the `report_cell_number` to match with the config. Also for the user reading this, they'll have more context on what a `report_cell_number` is before reading `cell_number_type`.

Reply:
@mwakaba2 Thank you for your comment. 
Yes, I agree with you. But I have some difficulty.
These setting examples are copied from the actual default settings of my development version.
It seems that the setting entries are automatically reordered by alphabetic order (I also saw the other settings.)
So, I can move the position of `cell_number_type` just for README.md, but I can not move the actual entry position.

If it's OK just to fix the README, I'd like to fix it. Can I ask what you think about it?